Refs #26029 -- Minimally documented StorageHandler API. (#18277)
django/django: docs/ref/files/storage.txt

.. data:: storages

>     Mapping of aliases to storage instances as defined by :setting:`STORAGES`.

Reviewer: ```suggestion
    A dictionary-like object that allows you to retrieve a storage instance
    using its alias as defined by :setting:`STORAGES`.
```
This is roughly the wording used for connections. I think I like "A dictionary-like object" more than "Mapping" here

Author: Yep, no problem. 👍

.. data:: storages

>     A dictionary-like object that allows you to retrieve a storage instance

Reviewer 2: I believe avoiding the use of pronouns like "you" makes the documentation more direct and objective, as it emphasizes the capabilities of a program/code rather than addressing the reader.
```suggestion
    A dictionary-like object that allows to retrieve a storage instance
```

Author: I'd either go `allows retrieving` or drop the allows altogether and just have `to retrieve`. As you think.

    using its alias as defined by :setting:`STORAGES`.

>     ``storages`` has a dictionary attribute ``backends``, which at startup is

Reviewer 2: Since we already said `storages` is a dict-like (but not **necessarily** a dict), I would be less prescriptive here by saying "has a key" (which is really what the code cares about, no?).
Also, I'm not sure what "startup" is... is it on server first start? reload? all the above? How about something like "on initial settings load" or even the simpler (but more vague, so that's a con) "defaults"?
```suggestion
    ``storages`` has a key ``backends``, which defaults to
```

Author: It's not a key per se... 

You do storage lookup via dictionary (key) access `storages["compressor"]`  but access backends as `storages.backends`.

Reviewer 2: Thanks for the clarification, then I guess we could say just "attribute".

>     the raw value provided to :setting:`STORAGES`.

Reviewer 2: I know I'm the non-native English speaker in this reviewer set but this reads odd to me, I would suggest:
```suggestion
    the raw value provided in :setting:`STORAGES`.
```

Author: 🤷‍♀️ either work for me. As you think is clearer 👍

>     Additionally ``storages`` provides a ``create_storage()`` method, accepting
>     the dictionary used in :setting:`STORAGES` for the backend, that will
>     return a storage instance. This may be useful for third-party packages
>     needing to instantiate storages in tests::

Reviewer 2: Slight rephrasing, and adding the necessary comma after the "Additionally" introductory word:
```suggestion
    Additionally, ``storages`` provides a ``create_storage()`` method that
    accepts the dictionary used in :setting:`STORAGES` for the backend, and
    returns a storage instance. This may be useful for third-party packages
    needing to instantiate storages in tests::
```

Author: Yep, fine. 👍

.. data:: storages

    Mapping of aliases to storage instances as defined by :setting:`STORAGES`.

    ``storages`` has a dictionary attribute ``backends``, which at startup is
    the raw value provided to :setting:`STORAGES`.

    Third-party packages providing storages may allow users to optionally
    customize a storage for a known alias, using ``backends`` to provide a

>     default::

Reviewer 3: ```suggestion
    default. Here's an example implementation of such an utility::
```

Maybe that's superfluous. As a reader it wasn't immediately clear to me that `get_storage` is a thing which should be added to a third party package. The additional clarification may be helpful.

Also, I learned another thing today :-)

Reviewer: I agree, I am feel like this example should be over here: https://docs.djangoproject.com/en/5.1/howto/custom-file-storage/#use-your-custom-storage-engine
